opt/main.py

```diff
@@ -3,33 +3,71 @@
 """
 
 import os
+from typing import Union
 
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 
 from scopeserver.config import from_file
-from scopeserver import message_of_the_day, SCopeServer
+from scopeserver import message_of_the_day, LegacySCopeServer, SCopeServer
+from scopeapi import SCopeAPI
+from scopeapi.model import Error
+from scopeapi.model import GetUUID
+from scopeapi.model import GetRemainingUUIDTime, GetDatasets
 
 scope_api = FastAPI()
+scope_api.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
 CONFIG = from_file(os.environ.get("SCOPE_CONFIG"))
 
+__scope_server = SCopeServer(config=CONFIG)
+__scope_server_legacy = LegacySCopeServer(config=CONFIG, new_server=__scope_server)
+__scope_api = SCopeAPI(server=__scope_server)
```

Comment:
Is there any reason you're using a dunder prefix here? I would prefer you avoid it.

Reply:
I will change that